src/d3a/models/area/redis_dispatcher/area_event_dispatcher.py

```diff
@@ -0,0 +1,54 @@
+import json
+from random import random
+from d3a.events import AreaEvent
+from d3a.models.area.redis_dispatcher import RedisEventDispatcherBase
+
+
+class RedisAreaEventDispatcher(RedisEventDispatcherBase):
+    def __init__(self, area, root_dispatcher, redis):
+        super().__init__(area, root_dispatcher, redis)
+        self.str_area_events = [event.name.lower() for event in AreaEvent]
+
+    def event_channel_name(self):
+        return f"{self.area.slug}/area_event"
```

Comment:
There is a check at the launch of [simulation](https://github.com/gridsingularity/d3a/blob/master/src/d3a/d3a_core/simulation.py#L163) to be sure that area names should be unique. Im wondering if it would still work or fail if some external agent tried to get connected to an already configured and launched d3a-sim with a non-unique name?

Reply:
Unfortunately in this case it is required to have unique area names in order for this to work. In the real world, this is easily circumvented by using the area.uuid instead of the area name, which we do already for the markets. Changing this, and using uuids instead. Thanks!